README.md

```diff
@@ -2,23 +2,46 @@
 
 The Boo Programming Language (c) 2009 Rodrigo B. de Oliveira (rbo@acm.org)
 
+Prerequisites
+=============
+
+## Mono
+
+If you are building on Mono, you need to have a compatible version from the 4.2.x line. The most recent compatible version is [4.2.4](https://download.mono-project.com/archive/4.2.4/).
+
+## Build Tools
+
+### Windows/Linux
+You'll need to install compatible versions of NAnt and NUnit. See [build-tools/versions](build-tools/versions) for known-good versions. _NAnt must be cloned and built from sources_; the last official release (0.92) does not work.
```

Comment:
Our of curiosity, what's the problem with official NAnt?

Also, if it works from their source repo but not from their latest release, how long is this likely to remain the case?